File: tuts/010-cloudmap-service-discovery/java/src/main/java/com/example/servicediscovery/GettingStartedScenario.java
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.servicediscovery;

import software.amazon.awssdk.services.servicediscovery.CloudMapClient;

public class GettingStartedScenario {
    public static void main(String[] args) {
        try (CloudMapClient client = CloudMapClient.builder().build()) {
            CloudMapWrapper wrapper = new CloudMapWrapper(client);
            System.out.println("Running CloudMap getting started scenario...");
            // TODO: setup, interact, teardown
            System.out.println("Scenario complete.");
        }
    }
}

File: tuts/010-cloudmap-service-discovery/kotlin/src/main/kotlin/com/example/servicediscovery/GettingStartedScenario.kt
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.servicediscovery

import aws.sdk.kotlin.services.servicediscovery.CloudMapClient
import kotlinx.coroutines.runBlocking

fun main() = runBlocking {
    CloudMapClient { region = "us-east-1" }.use { client ->
        val wrapper = CloudMapWrapper(client)
        println("Running CloudMap getting started scenario...")
        // TODO: setup, interact, teardown
    }
}
